bolt/lib/Core/Exceptions.cpp

```diff
@@ -640,16 +640,16 @@ bool CFIReaderWriter::fillCFIInfoFor(BinaryFunction &Function) const {
         BC.errs() << "BOLT-WARNING: DW_CFA_MIPS_advance_loc unimplemented\n";
       return false;
     case DW_CFA_GNU_window_save:
-      // DW_CFA_GNU_window_save and DW_CFA_GNU_NegateRAState just use the same
-      // id but mean different things. The latter is used in AArch64.
+      // DW_CFA_GNU_window_save and DW_CFA_AARCH64_negate_ra_state just use the
+      // same id but mean different things. The latter is used in AArch64.
       if (Function.getBinaryContext().isAArch64()) {
-        // Not adding OpNegateRAState since the location they are needed
+        // The location OpNegateRAState CFIs are needed
```

Comment:
I think you could drop 'are' here?

// The location OpNegateRAState CFIs ~~are~~ needed

Reply:
Grammar-wise I believe it is needed.